tests/qnode_new/test_qnode_jacobian.py

```diff
@@ -565,6 +583,25 @@ def circuit(x):
             assert grad_A == pytest.approx(grad_F, abs=tol)
             assert grad_A2 == pytest.approx(grad_F, abs=tol)
 
+    @pytest.mark.xfail(reason="FIXME: 'A' method fails on QuadOperator (it has no gradient recipe)", raises=AttributeError, strict=True)
```

Comment:
Separating the `Operator` and `Observable` classes a while ago created an issue where observables that depend on a parameter cannot be differentiated using the par-shift method, since they have no gradient recipe.

Reply:
Was this something that used to work? I suppose it still works if you stick a rotation before qml.X, but that's a bit annoying...

Reply:
It's quite possible it never worked, but it can and should. I think if you add a `grad_recipe` to `QuadOperator` and make a small change to `_best_method` the standard par-shift method can handle it.

In this particular case a single evaluation would be enough since the Heisenberg vector is just zeros, sines and cosines, but that'd require implementing a whole new method.

Reply:
Okay, we should make an issue to add in and test this behaviour (should be as simple as moving 'grad_recipe' and 'grad_method' from `Operation` to `Operator`).